SocialFilter/sections/specific.txt

```diff
@@ -2698,7 +2698,7 @@ manutd.com##div[class^="home-content-panel_"] > div.left > ul > li:not(.email):n
 gdpi.hitbullseye.com##.post-entry-share
 help.line.me##.answer_share
 sakura-checker.jp,rallys.online,virtualsan-looking.jp,lawson.co.jp,matsukiyo.co.jp##.snsList
-social-dog.net##.snslist
+doutor.co.jp,social-dog.net##.snslist
```

Comment:
I can't see too. because the element was commented out...
<details><summary>screenshot</summary>

![](https://i.ibb.co/4SqXb0y/image.png)
</details><br/>

Reply:
Ok)